app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk

```diff
@@ -286,7 +286,7 @@
         { heading: "Are there any risk management issues if working in a supervised group?", data: renderValue( answers["risk_management_issues_supervised_group"], answers["risk_management_issues_supervised_group_details"], "Yes" ) },
         { heading: "Alcohol or drug issues with health and safety impact?", data: renderValue( answers["alcohol_drug_issues"], answers["alcohol_drug_issues_details"], "Yes" ) } ] -%}
 
-    {%- if widgets.mappa.isNominal %}
+    {%- if widgets.mappa.isNominal != null %}
```

Comment:
I think `isNominal` will be `true` if the  level code is `M0`, `false` if the level code is anything else, and `null` if the field is missing. So this will show regardless of if it's nominal. Just making sure I understand 😅

Reply:
I think that's correct - the rules around this have been very woolly but will throw this to test to verify with UCD.